Samples/GStreamer/Program.cs

```diff
@@ -1,10 +1,10 @@
-﻿namespace Samples
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Sample.Gst.Play();
-        }
-    }
-}
+﻿namespace Samples
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Sample.Gst.Play();
```

Comment:
This all looks good, but I've noticed `Sample.Gst.Play()` takes a while to startup for me. Could we get a `Console.WriteLine` here just to indicate that the program has started?

Reply:
I put the `Console.WriteLine` into the `Play` method, like in the DBus samples.